registration-processor/registration-processor-packet-manager/src/main/java/io/mosip/registration/processor/packet/manager/decryptor/DecryptorImpl.java

```diff
@@ -232,10 +204,10 @@ public InputStream decrypt(InputStream packetStream, String registrationId)
 					: description.getCode();
 			String moduleName = ModuleName.DECRYPTOR.toString();
 			auditLogRequestBuilder.createAuditRequestBuilder(description.getMessage(), eventId, eventName, eventType,
-					moduleId, moduleName, registrationId);
+					moduleId, moduleName, refId);
```

Comment:
In audit we cannot send the refId as registration Id, we should have a different method without the registration id or pass the registration id addionally to the decrypt method